src/main/resources/swagger/api-docs.yaml

```diff
@@ -263,6 +263,61 @@ paths:
           description: Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
+  '/notebooks/{googleProject}/{clusterName}/api/localize':
+    post:
+      summary: Localize files to/from a Jupyter notebook server
+      description: |
+        Sends a command to a Jupyter notebook server to localize files between the server and GCS.
+        This operation will happen asynchronously; output, including any errors, will appear in `localization.log` in the working directory of the Jupyter notebook server.
+      operationId: proxyLocalize
+      tags:
+        - notebooks
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          type: string
+        - in: path
+          name: clusterName
+          description: clusterName
+          required: true
+          type: string
+        - in: body
+          description: |
+            JSON object. Keys represent destinations, values represent sources.
+            Paths will be quoted, intermediate local directories made, and then sent to [gsutil cp](https://cloud.google.com/storage/docs/gsutil/commands/cp).
+            Note that duplicate keys will lead to unexpected behaviour, so specify the destination filename explicitly if you want to localize multiple files to the same directory.
+          name: filesToLocalize
+          required: true
+          schema:
+            type: object
+            description:
+            example:
+              "/local/file/system/file.txt" : "gs://somebucket/file.txt"
+              "/localize/entire/directory" : "gs://somebucket/*"
+              "gs://upload/to/this/bucketdir" : "/this/local/file.txt"
+      responses:
+        '200':
+          description: Proxy connection successful
+        '400':
+          description: "Bad request. Your POST body is probably malformed: it should be a string/string JSON object."
```

Comment:
Extra quotes here?
Also needs 
```
  schema:
    $ref: '#/definitions/ErrorReport'
```

Reply:
It actually _needs_ the quotes because otherwise the colon confuses the yaml parser.

Reply:
And it doesn't return an ErrorReport because this status comes from the extension, not Leo.

Reply:
god i hate yaml